users/app.py

```diff
@@ -60,5 +54,43 @@ def add_update_user():
     return jsonify(user_object), (201 if added_new_user else 200)
 
 
+def find_authorization_in_db(username, users_collection):
+    """Queries the db to find authorization of the given user.
+
+    Documents in the users collection should look like
+        {"username": "cmei4444",
+        "name": "Carolyn Mei",
+        "is_organizer": True}
+    """
+    if users_collection.count_documents({"username": username}) == 0:
+        return False  # user not found
+    cursor = users_collection.find({"username": username})
+    first_user = cursor[0]
+    authorized = first_user.get("is_organizer")
+    return False if authorized is None else authorized
+
+
+@app.before_first_request
+def initialize_mongodb():
+    """Connect to MongoDB Atlas database using env vars.
+
+    app.before_first_request decorator is used to keep
+    the DB from initializing in testing to remove the
+    dependency on the MongoDB connection in tests.
+    """
+    mongodb_atlas_username = os.environ.get(
```

Comment:
Since these are local variables, and the name of the function is "...mongodb", you can probably use shorter names here, like `username`, `password`, and `cluster`.

Reply:
Done https://github.com/knative-portability/large-events/pull/2/commits/1676bea8fdd14a825b2efc42215a85e9c2b18571

Reply:
Actually, just changed the whole function to be simpler https://github.com/knative-portability/large-events/pull/2/commits/b943855d9abb766fafbe7e91de3bd0e173ce491d